changelog.d/pr-10113

```diff
@@ -0,0 +1,10 @@
+synopsis: Canonilise path passed via `--with-compiler`.
```

Comment:
I think the ancient verb is `canonicalize` and the same below. :)

Reply:
```suggestion
synopsis: Canonicalise path passed via `--with-compiler`.
```